Makefile

```diff
@@ -45,10 +62,27 @@ LICENSE_HEADER:
 
 .PHONY: license
 license: LICENSE LICENSE_HEADER
-	cd client && $(MAKE) license
-	cd node && $(MAKE) license
+	cd client && npm ci && npm run license:fix
+	cd docker && npm ci && npm run license:fix
 
 .PHONY: test-license
 test-license: LICENSE LICENSE_HEADER
 	cd client && $(MAKE) test-license
```

Comment:
I'd like us to move in the direction of having the license rules live and run just on place (in the project root), but we can defer that for this PR